codesign/test/file_codesing_visitor_test.dart

```diff
@@ -11,18 +11,19 @@ import './src/fake_process_manager.dart';
 
 void main() {
   const String randomString = 'abcd1234';
-  late MemoryFileSystem fileSystem;
-  fileSystem = MemoryFileSystem.test();
-  Logger logger = Logger('codesign-test');
+  cs.logger = Logger('codesign-test');
```

Comment:
Tests are able to reuse the singleton from the lib/ code, no need to create another here

Reply:
Sure, removed.